jdaviz/configs/cubeviz/plugins/sonify_data/sonify_data.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <j-tray-plugin
+    :description="docs_description"
+    :link="docs_link || 'https://jdaviz.readthedocs.io/en/'+vdocs+'/'+config+'/plugins.html#cubeviz-sonify-data'"
+    :uses_active_status="uses_active_status"
+    @plugin-ping="plugin_ping($event)"
+    :keep_active.sync="keep_active"
+    :popout_button="popout_button"
+    :scroll_to.sync="scroll_to">
+
+    <j-plugin-section-header>Sonify Cube</j-plugin-section-header>
+    <v-alert v-if="!has_strauss" type="warning" style="margin-left: -12px; margin-right: -12px">
+      To use Sonify Data, install strauss and restart Jdaviz. You can do this by running `pip install .[strauss]`
+      in the command line and then launching Jdaviz.
+    </v-alert>
```

Comment:
can use the `disabled_msg` traitlet instead and then the rest of the plugin will automatically not be shown without having to `v-else` everything else.

Reply:
We would lose the yellow alert box but that's fine with me.

Reply:
This doesn't make clear to the user _where_ they need to run this command. It won't work anywhere but the root `jdaviz` directory. It also doesn't apply if the user installed jdaviz from pip rather than locally (in that case, `pip install jdaviz[strauss]` should work, I believe. Maybe the best thing to do here is link to a docs section that has a more detailed explanation.